src/components/LoginForm/index.js

```diff
@@ -32,30 +49,48 @@ const LoginForm = ({ onSubmit }) => {
 
   const inputProps = useTextInputProps(handleValueChange, handleBlur, values);
 
+  const buttonTitle = useMemo(
+    () => (status === LOADING ? strings.COMMON.loading : strings.SIGN_IN.button),
+    [status],
+  );
+
+  const additionalEmailStyles = useMemo(
+    () => (error || errors.email ? INPUTBORDERERROR : INPUTBORDER),
+    [error, errors],
+  );
+
+  const additionalPasswordStyles = useMemo(
```

Comment:
I think these memoizations could be more expensive than the computation they contain. Remember `memo` is not free.